packages/calcite-components/src/components/date-picker/date-picker.e2e.ts

```diff
@@ -246,8 +247,8 @@ describe("calcite-date-picker", () => {
       element.setProperty("max", undefined);
       await page.waitForChanges();
 
-      expect(await element.getProperty("minAsDate")).toBe(undefined);
-      expect(await element.getProperty("maxAsDate")).toBe(undefined);
+      expect(await element.getProperty("minAsDate")).toMatchObject({});
+      expect(await element.getProperty("maxAsDate")).toMatchObject({});
```

Comment:
I don't think this will be applicable based on ☝️, but for future reference, this should assert on a Date instance, value or format.